hedera-node/hedera-smart-contract-service-impl/build.gradle.kts

```diff
@@ -31,4 +31,9 @@ configurations.all {
 dependencies {
     api(project(":hedera-node:hedera-smart-contract-service"))
     implementation(project(":hedera-node:hedera-mono-service"))
+    testImplementation(project(mapOf("path" to ":hedera-node:hedera-mono-service")))
```

Comment:
What is this doing?

Reply:
I think I added it for some dependency. Removed it, as it is not needed anymore